data-avro/src/main/java/com/linkedin/data/avro/util/AvroUtil.java

```diff
@@ -30,16 +30,22 @@
 public class AvroUtil
 {
   public static String jsonFromGenericRecord(GenericRecord record) throws IOException
+  {
+    return jsonFromGenericRecord(record, true);
+  }
+
+  public static String jsonFromGenericRecord(GenericRecord record, boolean pretty) throws IOException
   {
     GenericDatumWriter<GenericRecord> writer = new GenericDatumWriter<>();
     ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-    Encoder jsonEncoder = AvroCompatibilityHelper.newJsonEncoder(record.getSchema(), outputStream, true);
+    Encoder jsonEncoder = AvroCompatibilityHelper.newJsonEncoder(record.getSchema(), outputStream,pretty);
```

Comment:
the json format has changed between avro 1.4 and 1.6. there's a variant of this method that allows you to specify the version of avro (determines the json format) - i'd set it explicitly

Reply:
added the util method which takes AvroVersion